src/main/java/com/webcohesion/ofx4j/domain/data/tax1099/Tax1099B.java

```diff
@@ -114,7 +114,7 @@ public void setPayerId(String payerId) {
 	/**
 	 * @return the recAddress
 	 */
-	@ChildAggregate(required=true, order = 5)
```

Comment:
![image](https://user-images.githubusercontent.com/12237604/72668336-9baf1400-39f3-11ea-90c1-5021a07a1bfa.png)

RECADDR isn't required according to the spec